extensions/mssql/src/tableDesigner/tableDesigner.ts

```diff
@@ -6,21 +6,23 @@
 import { AppContext } from '../appContext';
 import * as azdata from 'azdata';
 import * as vscode from 'vscode';
-import { sqlProviderName, TableType } from '../constants';
+import { sqlProviderName } from '../constants';
 import { generateUuid } from 'vscode-languageclient/lib/utils/uuid';
 import { ITelemetryEventProperties, Telemetry } from '../telemetry';
 
 export function registerTableDesignerCommands(appContext: AppContext) {
 	appContext.extensionContext.subscriptions.push(vscode.commands.registerCommand('mssql.newTable', async (context: azdata.ObjectExplorerContext) => {
 		const connectionString = await azdata.connection.getConnectionString(context.connectionProfile.id, true);
-		const telemetryInfo = await getTelemetryInfo(context, TableType.Basic);
+		const tableSubType = context.nodeInfo.nodeSubType ? context.nodeInfo.nodeSubType : azdata.designers.TableSubType.Temporal;
```

Comment:
You can simplify this to `context.nodeInfo.nodeSubType ?? azdata.designers.TableSubType.Temporal`

But why are you defaulting to temporal? Most tables aren't going to be any of the sub types

Reply:
All the tables are temporal right now

Reply:
What does that mean? What tables? 

And if it's actually a temporal table then why isn't the nodesubtype already set to that? An empty string is a perfectly valid value for this field currently...